release-team/release-team-removal.md

```diff
@@ -0,0 +1,48 @@
+# Resignation
```

Comment:
```suggestion
# Release Team Removal Process

The release team is reassembled with a new team for each cycle. All team members are automatically removed if they do not apply and are selected for the next release cycle. This process works well, but from time to time a team member may leave the release team in the middle of the release cycle or the release team leaders may decide to part ways early. This document describes the process for this eventuality.

## Resignation
```

Reply:
lets add some intro to the document.

Reply:
I agree- I think an intro would be helpful. Maybe rephrase slights: 

```
All team members are automatically removed if they do not re-apply and are selected for the next release cycle. While this process generally functions smoothly, there are instances when a team member might leave mid-cycle or the release team leaders may decide to part ways early. 
```